x/oracle/keeper/keeper.go

```diff
@@ -172,21 +156,18 @@ func (k Keeper) ProcessUpdateWhiteListValidator(ctx sdk.Context, cosmosSender sd
 // power to be considered successful, or alternatively,
 // will never be able to become successful due to not enough validation power being
 // left to push it over the threshold required for consensus.
-func (k Keeper) processCompletion(ctx sdk.Context, prophecy types.Prophecy) types.Prophecy {
-	highestClaim, highestClaimPower, totalClaimsPower, totalPower := prophecy.FindHighestClaim(ctx, k.stakeKeeper, k.GetOracleWhiteList(ctx))
-
-	highestConsensusRatio := float64(highestClaimPower) / float64(totalPower)
-	remainingPossibleClaimPower := totalPower - totalClaimsPower
-	highestPossibleClaimPower := highestClaimPower + remainingPossibleClaimPower
-	highestPossibleConsensusRatio := float64(highestPossibleClaimPower) / float64(totalPower)
-
-	if highestConsensusRatio >= k.consensusNeeded {
+func (k Keeper) processCompletion(ctx sdk.Context, networkDescriptor types.NetworkDescriptor, prophecy types.Prophecy) types.Prophecy {
```

Comment:
Why did we change this use vetoRatio?

Reply:
it is much easier to judge if the prophecy was failed